api/src/controllers/cards.controller.ts

```diff
@@ -61,7 +63,7 @@ export class CardsController {
 
       const findAllCardsData: PokemonCard[] = await this.card.findAllCards(page, limit, searchByName, cardName);
 
-      res.status(200).json({ data: findAllCardsData, message: 'findAll' });
+      res.status(200).json({ data: findAllCardsData, message: 'findAll', totalPages: totalNumberOfPages });
```

Comment:
This `totalNumberOfPages` variable will not be able to be read here because it is out of the scope of where it was declared (it made an error come up for me). I would declare the `totalNumberOfPages` variable before the if statements (i.e, around line 41): `let totalNumberOfPages: number; `. Then, change the initializations of `totalNumberOfPages` to just variable assignments: `totalNumberOfPages = . . .`

Reply:
I think this is a mistake with the merging because I made this before the most recent Pull Request. Meaning my master branch was outdated. But you are correct the totalNumberOfPages should be declared before the if statement.